nautobot/ipam/factory.py

```diff
@@ -362,7 +346,6 @@ class Meta:
     )
 
     has_role = factory.Faker("pybool")
-    role = factory.Maybe("has_role", random_instance(Role), None)
```

Comment:
this needs to change to use Roles from the extras factory, instead of being removed.